examples/chef/sumologic-otel-collector/resources/default.rb

```diff
@@ -27,7 +27,7 @@
   install_command = get_install_script_command(new_resource)
   execute INSTALL_SCRIPT_PATH do
     command install_command
-    environment ({"SUMOLOGIC_INSTALL_TOKEN" => resource.install_token})
+    environment ({"SUMOLOGIC_INSTALL_TOKEN" => new_resource.install_token})
```

Comment:
We renamed install token to installation token, so the final change should be:

```suggestion
    environment ({"SUMOLOGIC_INSTALLATION_TOKEN" => new_resource.installation_token})
```